modules/frus-html.xqm

```diff
@@ -695,7 +695,7 @@ declare function fh:exists-ebook($vol-id) {
 };
 
 declare function fh:exists-pdf($vol-id) {
-    exists(doc(concat($config:HSG_S3_CACHE_COL, 'frus/', $vol-id, '/ebook/resources.xml'))//filename[ends-with(., '.pdf')])
+    exists(doc(concat($config:HSG_S3_CACHE_COL, 'frus/', $vol-id, '/pdf/resources.xml'))//filename[ends-with(., '.pdf')])
```

Comment:
@joewiz I just wanted to sense check this with you: I believe it's in error, but let me know if it's deliberate!

Reply:
@yamahito Your change is correct, thanks!